tests/UI_WindowViewModelBase_Tests.cpp

```diff
@@ -21,30 +24,6 @@ TEST_CLASS(UI_WindowViewModelBase_Tests)
         WindowViewModelHarness() noexcept = default;
     };
 
-    class MockDesktop : public IDesktop, ra::services::ServiceLocator::ServiceOverride<IDesktop>
-    {
-    public:
-        MockDesktop() : ra::services::ServiceLocator::ServiceOverride<IDesktop>(this)
-        {
-        }
-
-        void ShowWindow(WindowViewModelBase& vmViewModel) const override
-        {
-            vmShownWindow = &vmViewModel;
-        }
-
-        ra::ui::DialogResult ShowModal(WindowViewModelBase& vmViewModel) const override
-        {
-            vmModalWindow = &vmViewModel;
-            return DialogResult::OK;
-        }
-
-        void Shutdown() override {}
-
-        mutable WindowViewModelBase* vmShownWindow = nullptr;
-        mutable WindowViewModelBase* vmModalWindow = nullptr;
-    };
```

Comment:
extracted into a separate file, merged with second MockDesktop in VM_MessageBoxViewModel_Tests.cpp